src/components/home-page-slide-form/home-page-slide-form.js

```diff
@@ -0,0 +1,346 @@
+      link: slide.link || '',
+      show: slide.show || false,
+      order:slide.order||slideOrder
+    },
+
+    onSubmit: () => {
+      (()=>{
+        if(values.show&&slide.show){
+          values.order = slide.order
+          return
+        }
+        if(values.show){
+          values.order = slideOrder
+          return
+        }
+        if(!values.show){
+          values.order = 0
+        }
+      })()
+      const newSlide = createSlide(values);
+
+      if(id&&upload.name){
+        dispatch(updateSlide({id,slide:newSlide,upload}))
+        return
+      }
+      if(id){
+        dispatch(updateSlide({id,slide:newSlide}))
+        return
+      }
+      dispatch(addSlide({ slide: newSlide, upload }));
+    }
+  });
+  const checkboxes = [
+    {
+      id: 'show',
+      dataCy: 'show',
+      value: values.show,
+      checked: values.show,
+      color: 'primary',
+      label: config.labels.homePageSlide.show,
+      handler: () => setFieldValue('show', !values.show)
+    }
+  ];
+  const handleImageLoad = (e) => {
+    if (e.target.files && e.target.files[0]) {
+      const reader = new FileReader();
+      reader.onload = (event) => {
+        setFieldValue('slideImage', event.target.result);
+        setSlideImage(event.target.result);
+      };
+      reader.readAsDataURL(e.target.files[0]);
+      setUpload(e.target.files[0]);
+    }
+  };
+
+  return (
+    <div className={styles.formContainer}>
+      <form onSubmit={handleSubmit}>
+        <CheckboxOptions options={checkboxes} />
+
+        <Grid item xs={12}>
+          <Paper className={styles.slideItemUpdate}>
+            <span className={styles.imageUpload}>
+              {config.labels.homePageSlide.image}
+            </span>
+            <ImageUploadContainer
+              handler={handleImageLoad}
+              srcForAvatar={
+                slideImage ||`${config.IMG_URL}${slide.images.large}`
+              }
+              fileName={upload.name || slide.images.large}
+            />
+            <TextField
+              data-cy='link'
+              id='link'
+              className={styles.textField}
+              variant='outlined'
+              label={config.labels.homePageSlide.link}
+              value={values.link}
+              onChange={handleChange}
+              error={touched.link && !!errors.link}
+            />
+            {touched.link && errors.link && (
+              <div data-cy='material-error' className={styles.inputError}>
+                {errors.link}
+              </div>
+            )}
+          </Paper>
+        </Grid>
+        <AppBar position='static'>
+          <Tabs
+            className={styles.tabs}
+            value={tabsValue}
+            onChange={handleTabsChange}
+            aria-label='simple tabs example'
+          >
+            {languageTabs}
+          </Tabs>
+        </AppBar>
+        {languages.map((lang, index) => (
+          <TabPanel key={index} value={tabsValue} index={index}>
```

Comment:
Don't use `index` as a value for a key attribute, you can combine different fields to generate it, like this: `${lang}-${index}`

Reply:
Fixed